Add email with virtual competition details (#52)
srobo/team-emails: SR2020/2020-05-26-virtual-competition-details.md

---
to: sr2020-teams
subject: Student Robotics 2020 Virtual Competition Details
---

> ![](https://pbs.twimg.com/media/EYt9wgcX0AAKjOn)

Reviewer: What is this?

> ![](https://pbs.twimg.com/media/EYt9wgcX0AAKjOn?format=jpg)

Reviewer: Should probably embed the pic from the docs instead

Hello!

> Today's the day we announce some awaited details of the virtual competition for Student Robotics 2020.

Reviewer: ```suggestion
Today's the day we announce some long awaited details of the virtual competition for Student Robotics 2020.
```

> Today's the day we announce some long awaited details of the virtual competition for Student Robotics 2020.

Reviewer: ```suggestion
Today's the day we announce some long-awaited details of the virtual competition for Student Robotics 2020.
```

## Simulator

> For the virtual competition, we've created a new simulator, based on [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available on the docs.

Reviewer: ```suggestion
For the virtual competition, we've created a new simulation, based in [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available in the docs.
```
Because of Peter's comment that we haven't actually made the simulator, just a project inside it

[Competition Simulator Docs](https://studentrobotics.org/docs/competition-simulator/)

> Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator.

Reviewer: Maybe needs a "but we have tried to keep it similar enough that you can easily adapt what you've already written" or something along those lines?

> Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator, however we've tried to keep it as similar as possible, and easily adaptable from what you've already written.

Reviewer 2: We should probably clarify that the online IDE is not going to work well for developing for the virtual competition. Not sure if that needs to be in this email, but we do need to mention it somewhere (and obviously so). This paragraph is talking about the changes, so feels like it might fit.
Regardless of whether it's mentioned here, we definitely do need to mention it in the docs.

Author: I'm going to er towards the docs, as they're more likely to be read directly by students

> Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator, however we've tried to keep it as similar as possible, and easily adaptable from what you've already written.

Reviewer: ```suggestion
Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator. However, we've tried to keep it as similar as possible, and easily adaptable from what you've already written.
```

## Competition Format

> Our intention is to run this in a very similar way to the regular competition. In around early July, we'll run virtual competition events, taking place on weekends and streamed live. There will be a few league match sessions, followed by a knockout final. Your code will be submitted to us before the weekend session, ready for running. In between sessions, you are free to make any and all modifications to your robot's code.

Reviewer: ```suggestion
Our intention is to run this in a very similar way to the regular competition. In early July (subject to change), we'll run virtual competition events, taking place on weekends and streamed live. There will be a few league match sessions, followed by a knockout final. Your code will be submitted to us before the weekend session, ready for running. In between sessions, you are free to make any and all modifications to your robot's code.
```
Not sure about this one but it sounded a bit weird

> Our intention is to run this in a very similar way to the regular competition. In early July (subject to change), we'll run virtual competition events, taking place on weekends and streamed live. There will be a few league match sessions, followed by a knockout final. Your code will be submitted to us before the weekend session, ready for running. In between sessions, you are free to make any and all modifications to your robot's code.

Reviewer 2: ```suggestion
Our intention is to run this in a very similar way to the regular competition. In early July (subject to change), we'll run virtual competition sessions, taking place on weekends and streamed live. There will be a few league match sessions, followed by a knockout final. Your code will be submitted to us before the weekend session, ready for running. In between sessions, you are free to make any and all modifications to your robot's code.
```

As I understand the plan we're not running a single event as such -- just the sessions as described. I suggest that pick a single term an stick to it.
Alternatively, let's clarify that an event is comprised of several sessions (or whatever the relationship is between the two concepts).

## Simulator

For the virtual competition, we've created a new simulation, based in [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available in the docs.

[Competition Simulator Docs](https://studentrobotics.org/docs/competition-simulator/)

Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator. However we've tried to keep it as similar as possible, and easily adaptable from what you've already written.

## Competition Format

Our intention is to run this in a very similar way to the regular competition. In early July (subject to change), we'll run virtual competition sessions, taking place on weekends and streamed live. There will be a few league match sessions, followed by a knockout final. Your code will be submitted to us before the weekend session, ready for running. In between sessions, you are free to make any and all modifications to your robot's code.

More details on the format will follow in the next few weeks on [our website](https://studentrobotics.org/events/sr2020/virtual-competition/).

> ## Signup

Reviewer: ```suggestion
## Sign-up
```
Hopefully this'll fix the spellcheck?
